website/docs/components/copy/snippet/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Copy Snippet
-description: A button that copies the text content of the button itself.
+description: A button that allows users to copy a snippet of code
```

Comment:
@andgen404 I think this doesn't differentiate it enough from the copy/button.

Reply:
I can't think of any other descriptions for the Copy Snippet 😓 to me the first one made sense and this one also makes sense. My last thought would be:

A button that has the functionality to copy its own text content.

Reply:
@MelSumner the fact that it copies the text from its own text content is irrelevant for the consumers (and the end users). It's an implementation detail. We should describe what the component does, not how it's implemented internally.

Since the description of the `Copy::Button` is  `A button that copies the attached or associated text upon interaction.` I think there is enough differentiation.